web/html/src/manager/systems/bootstrap/bootstrap-minions.js

```diff
@@ -198,11 +236,43 @@ class BootstrapMinions extends React.Component {
                     </div>
                 </div>
                 <div className="form-group">
+                    <label className="col-md-3 control-label">Authentication method:</label>
```

Comment:
Maybe, as all the labels are upper-cased in this form, should we also uppercase this one as `Authentication Method:`?

Reply:
Done.